src/runtime/Module.h

```diff
@@ -34,6 +34,13 @@ class JITModule;
 
 struct WASMParsingResult;
 
+enum JITFlagValue : uint32_t {
+    useJIT = 1 << 0,
+    JITVerbose = 1 << 1,
+    JITVerboseColor = 1 << 2,
+    perf = 1 << 3
```

Comment:
What is this flag `perf`? And this flag name looks quite different from other flags.

Reply:
The `perf` flag indicates, that perf is enabled (it is for JITDump file generating) with `--perf` switch.

Reply:
Shall we change its name? `enableJITDump`?

Reply:
> Shall we change its name? enableJITDump?

Yes, that would be more clear.